app/templates/not-found.hbs

```diff
@@ -1,3 +1,15 @@
-<div class="ui container">
```

Comment:
Please have two different templates. One for not-found and one for page.

Reply:
@sumedh123 you have again used the same template for both.

Reply:
@niranjan94 I have used. I guess you saw old changes somehow.

Reply:
@sumedh123 could you check? Because I'm still seeing the one where pages.hbs is used for both not found error and page content.

Reply:
@niranjan94 How can we use separate templates then? I mean one way is create two components and then simple if statement, but we want to use the template of  route ```not-found```.